text/concepts.tex

```diff
@@ -371,7 +372,8 @@ \section{Building memory safety}
 \section[label=interrupt_sentries]{Controlling interrupt status with sentries}
 
 In conventional RISC-V (and most other architectures) the interrupt status is controlled via a special register.
-This register can be modified only in some privileged mode.
+This register can be modified only in a privileged mode.
+\amanda{The word "some" here is confusing.  The next sentence makes it sound like it's a specific privileged mode, so I'm making the change to reflect that, but I am not 100% confident that it's correct.  I'd actually recommend re-writing both sentences to be something like "This register can be modified only by code running with the Access System Registers permission..." but, you know, accurate.}
```

Comment:
It is a bit confusing because RISC-V has different privilege modes and the one that can change them is different depending on a bunch of things.  I think this edit is good though: the reader doesn't need to know or care about the weirdness of RISC-V.